### PR TITLE
Lint and typings -> @types

### LINKED ISSUE #1434: Project templates are using the pre-@types type definitions
The project templates are bundling their own .d.ts files. This causes issues when installing one of the new `@types` packages. For example:

1. Create a new TypeScript Express Nodejs app using the template.
2. Install another package, such as mongodb.
3. Install the types for that package (e.g. `npm install --save-dev @types/mongodb`).
4. Build the project.

Note that there are a bunch of errors as the `@types` package pulls in some duplicate declarations.

### PR DESCRIPTION
**Pull request by aozgaa**
Fixes #1434

### CODE REVIEW

**Comment by paulvanbrenk** [comment]
Did you make sure this works on Dev14 Update 3?

**Comment by aozgaa** [comment]
Works with both dev14 update 3 and dev15.

File: Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts [hunk 1]
```diff
@@ -28,10 +28,10 @@ app.use(function (req, res, next) {
 // will print stacktrace
 if (app.get('env') === 'development') {
     app.use((err: any, req, res, next) => {
-        res.status(err['status'] || 500);
+        res.status(err.status || 500);
         res.render('error', {
+            error: err,
```
Comment (anubmat) [inline Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts]: Are these changes some minor improvements, unrelated to the rest of the template changes?

Comment (aozgaa) [inline Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts]: yes. I ran the templates through tslint/eslint respectively with moderately aggressive rules.

In this particular case, I believe it was a matter of alphabetizing object literal members.

**Comment by anubmat** [comment]
👍

**Pull request merged**